spec/spec_helper.rb

```diff
@@ -66,3 +66,8 @@
   #     end
   config.infer_spec_type_from_file_location!
 end
+
+# this disables the new FactoryBot 5 behavour where FactoryBot.build(...) no longer
+# builds associations to other factories causing many of the section spect tests to
+# fail as the video_paper association is nil
+FactoryBot.use_parent_strategy = false
```

Comment:
This seems like something we should try get rid of. With this false it means 'building' a model will 'create' its references.  That has always been a poor behavior of FactoryBot. 
I'd guess those cases where the tests are failing could be fixed by changing some of the relationship modeling so the models don't have to be created in order for them to work properly. 

I'm not saying you should fix it now. But probably would be a good thing to fix before going to rails 5.

Reply:
We only really use `build` in that spec and `create` most everywhere else.  The `build` specs are checking for lots of invalid cases so we could switch to `create` and then change the asserts to expect exceptions on the save.

Reply:
What I was thinking was that with the correct modeling even with a build, the video_paper association will not nil. So then the test don't have to change, and the code is more robust.